dcos_test_utils/dcos_api.py

```diff
@@ -259,15 +259,22 @@ def _wait_for_dcos_history_data(self):
         ro = self.get('/dcos-history-service/history/last')
         # resp_code >= 500 -> backend is still down probably
         if ro.status_code <= 500:
-            log.info("DC/OS History is probably getting data")
```

Comment:
I wonder what the original rational was here. The return code is smaller than 500 and that means that "DC/OS History is probably getting data"? That does not make sense to me as it somehow seems to mix correlation and causation.

Anyway, with this patch you drag this log message along and it's now logged in less scenarios than before. Without knowing the original rationale, it's probably safe to assume that this log message makes (even?) less sense than before.

Reply:
(I vote for removing it while we're here).

Reply:
I'm ambivalent and so I will remove this as you feel strongly, and I don't have any ideas for a more precise description of the state. @mellenburg is of course welcome to describe what he'd like logged.

Reply:
I'm totally cool with removing the statement in favor of the cleaner "Waiting for DC/OS history, resp code is: "